cg/meta/backup/backup.py

```diff
@@ -114,6 +114,7 @@ def _process_flow_cell(
 
             self.extract_flow_cell(decrypted_flow_cell, run_dir)
             self.create_rta_complete(decrypted_flow_cell, run_dir)
+            self.create_copy_complete(decrypted_flow_cell, run_dir)
```

Comment:
Given that the file is already present in many cases, should we really add another creation instance rather than debug why the existing place is not functioning in all instances? Not really familiar with the flow, though.

Reply:
I think it missing might just be an intended feature, as the backup has been dependant on the development of logic from Illumina's side. This simply complies with the logic that we have tried to create on out end to make sure that everything is is place once the decryption is complete.